### PR TITLE
Methods - Dimensionality Reduction

### PR DESCRIPTION
**Pull request by danich1**
This PR follows #5. Everything is now ready for review.

### CODE REVIEW

File: content/03.methods.md [hunk 1]
```diff
@@ -43,10 +43,13 @@ Following training, we generated a document vector for every article within bioR
 This document vector is calculated by taking the average of every token present within a given article, ignoring those that were absent from the word2vec model.
 
 #### Dimensionality Reduction of Document Embeddings
-1. Explain how tSNE works (paragraph one)
-2. Explain how PCA works  (paragraph two)
-3. Discuss how words were mapped onto PC components via cosine similarity
-4. ^ Explain cosine similarity
+We used t-Distributed Stochastic Neighbor Embedding (tSNE) [@raw:tsne_Maaten] to generate a global view of the bioRxiv preprint landscape.
```
Comment (cgreene) [inline content/03.methods.md:46]: I thought the plan was to use PCA for interpretability. Are you still planning to leave tSNE in as well?

File: content/03.methods.md [hunk 2]
```diff
@@ -43,10 +43,13 @@ Following training, we generated a document vector for every article within bioR
...
 #### Dimensionality Reduction of Document Embeddings
-1. Explain how tSNE works (paragraph one)
-2. Explain how PCA works  (paragraph two)
-3. Discuss how words were mapped onto PC components via cosine similarity
-4. ^ Explain cosine similarity
+We used t-Distributed Stochastic Neighbor Embedding (tSNE) [@raw:tsne_Maaten] to generate a global view of the bioRxiv preprint landscape. 
+This model was trained on bioRxiv document vectors using scikit-learn's [@raw:scikit-learn] implementation with a random seed of 100 and default settings for all other parameters.
+
+We used principal component analysis (PCA) [@doi:10.1111/1467-9868.00196] to project bioRxiv document vectors into a low dimensional space.
+This model was trained using scikit-learn's implementation with a random seed of 100, output of 50 principal components, and default settings for all other parameters.
```
Comment (cgreene) [inline content/03.methods.md:50]: Is this using a randomized solver?

File: content/03.methods.md [hunk 3]
```diff
@@ -43,10 +43,13 @@ Following training, we generated a document vector for every article within bioR
...
+For each principal component we calculated its cosine similarity with  all tokens in our word2vec model's vocab list.
```
Comment (cgreene) [inline content/03.methods.md:51]:
```suggestion
For each principal component we calculated its cosine similarity with  all tokens in our word2vec model's vocabulary.
```

File: content/03.methods.md [hunk 4]
```diff
@@ -43,10 +43,13 @@ Following training, we generated a document vector for every article within bioR
...
+We report the top 100 positive and negative scoring tokens in the form of  word clouds.
```
Comment (cgreene) [inline content/03.methods.md:52]: Does size in the cloud denote anything?

File: content/03.methods.md (hunk 1 above)
Comment (danich1) [reply content/03.methods.md:46]: Ah. I was under the impression to include it; however, now that I think about it sticking with PCA only is the better choice.

File: content/03.methods.md (hunk 2 above)
Comment (danich1) [reply content/03.methods.md:50]: Yup. The default is for PCA to figure out which algorithm is best. After reading the documentation, I found that scikit-learn's solver determined that randomized solver is most optimal.

File: content/03.methods.md [hunk 5]
```diff
@@ -43,10 +43,10 @@ Following training, we generated a document vector for every article within bioR
 This document vector is calculated by taking the average of every token present within a given article, ignoring those that were absent from the word2vec model.
 
 #### Dimensionality Reduction of Document Embeddings
-1. Explain how tSNE works (paragraph one)
-2. Explain how PCA works  (paragraph two)
-3. Discuss how words were mapped onto PC components via cosine similarity
-4. ^ Explain cosine similarity
+We used principal component analysis (PCA) [@doi:10.1111/1467-9868.00196] to project bioRxiv document vectors into a low dimensional space.
+This model was trained using scikit-learn's [@raw:scikit-learn] implementation with a random seed of 100, output of 50 principal components, and default settings for all other parameters.
```
Comment (cgreene) [inline content/03.methods.md:47]:
```suggestion
We trained this model using scikit-learn's [@raw:scikit-learn] implementation of a randomized solver [@arxiv:0909.4061] with a random seed of 100, output of 50 principal components, and default settings for all other parameters.
```

**Pull request merged**